.github/workflows/tests.yml

```diff
@@ -15,13 +14,14 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - name: Build
-      run: cargo build --verbose
+      run: cargo +${{ matrix.rust}} build --verbose
```

Comment:
nit: can we instead have a run step called "install rust" that installs rust at the right version and sets it as default?